Common/GPU/Vulkan/VulkanRenderManager.cpp

```diff
@@ -1625,10 +1645,153 @@ VKRPipelineLayout *VulkanRenderManager::CreatePipelineLayout(BindingType *bindin
 
 	vulkan_->SetDebugName(layout->descriptorSetLayout, VK_OBJECT_TYPE_DESCRIPTOR_SET_LAYOUT, tag);
 	vulkan_->SetDebugName(layout->pipelineLayout, VK_OBJECT_TYPE_PIPELINE_LAYOUT, tag);
+
+	for (int i = 0; i < VulkanContext::MAX_INFLIGHT_FRAMES; i++) {
+		layout->frameData[i].pool.Create(vulkan_, bindingTypes, (uint32_t)bindingTypesCount, 512);
```

Comment:
Is 512 enough?  Just asking because we've had to raise these numbers before.  I suppose it's for each pipeline... not really sure what the cost is here of increasing it.

Well, maybe it was 512 before...

-[Unknown]

Reply:
It's not for each pipeline, it's for each pipeline layout, and we only use two - one for UI and one for in-game.

1024 might be a more reasonable starting point, a lot of games end up there, while much fewer ones go all the way to 2048 and beyond (basically GTA).